tests/actor/test_resampling.py

```diff
@@ -26,7 +26,7 @@
 
 
 @pytest.fixture(autouse=True)
```

Comment:
You can remove the `autouse=True` too, as this magically replaces the event loop in all tests. Maybe add a comment instead so people doesn't get confused and think that adding or removing `autouse` have any effect?

Reply:
Right, I'll add a comment then